packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts

```diff
@@ -177,6 +179,24 @@ export class ModelViewerGLTFInstance extends GLTFInstance {
     }
 
     const clone = material.clone() as MeshStandardMaterial;
+    if (material.map != null) {
+      material.map = material.map.clone();
```

Comment:
This is how I tried to fix this the first time too, but it doesn't work. Cloning does not do the right thing and it ended up breaking things, see #1711 and #2279.

Reply:
Per conversation, shininess bug is not reproducible